rtc-video-quality/generate_data.py

```diff
@@ -453,18 +459,29 @@ def generate_jobs(args, temp_dir):
     jobs = []
     for clip in args.clips:
         bitrates = find_bitrates(clip['width'], clip['height'])
-        for bitrate_kbps in bitrates:
+        qp_values = find_qp()
+
+        if args.enable_bitrate:
+            params = bitrates
+        else:
+            params = qp_values
+        for param in params:
```

Comment:
This could just be:

for param in (find_bitrates(clip['width'], clip['height'])
if args.enable_bitrates else find_qp()):